lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  @include container-horizontal;
+  @include nx-truncate-ellipsis;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  height: 32px;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-md 6px $nx-spacing-md;
+  // Uppercase Min so SCSS ignores it and uses CSS min, not SCSS min
+  max-width: Min(100%, 320px);
+}
+
+.nx-tag--selectable {
+  align-items: center;
```

Comment:
Might not be necessary if https://github.com/sonatype/sonatype-react-shared-components/pull/290/files?file-filters%5B%5D=.scss&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r563776523 is implemented

Reply:
> Might not be necessary if https://github.com/sonatype/sonatype-react-shared-components/pull/290/files?file-filters%5B%5D=.scss&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r563776523 is implemented

Unfortunately that link doesn't resolve into anything useful for me.

Reply:
> Might not be necessary if https://github.com/sonatype/sonatype-react-shared-components/pull/290/files?file-filters%5B%5D=.scss&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r563776523 is implemented

I did lots of fiddling around with the positioning and Flexbox worked best.